src/locales/pt-br.ts

```diff
@@ -47,62 +47,71 @@ export default {
     GREEN: 'verde',
     PURPLE: 'roxo',
     RED: 'vermelho',
-    ROSE: 'rosa',
+    ROSE: 'rosa',  // TODO: ??? róseo
     WHITE: 'branco',
   },
+  ranks: {
+    SOLEMNITY: 'Solenidade',
+    SUNDAY: 'domingo',  // TODO: Is it always written with capital first letter or not?
+    TRIDUUM: 'Tríduo Pascal',
+    HOLY_WEEK: 'Semana Santa',
+    FEAST: 'festa',
+    MEMORIAL: 'memória',
+    OPT_MEMORIAL: 'memória facultativa',
+    COMMEMORATION: 'comemoração',  // TODO: Should it be rather `comemoração facultativa`?
+    WEEKDAY: 'ferial',
+  },
```

Comment:
I'm pretty sure for the 2 first one, but not for weekday as I found `ferial` in one document and `dia de semana` in another document.
From this [document](https://vdocuments.mx/normas-universais-do-ano-liturgico-e-o-novo-calendario-romano-geral-paulo-vi.html) (point 59) they use `dia de semana` so it might be better to use this translation.

Also a commemoration is always optional / facultative so it's no necessary to mention "facultativa" (we should pay attention to ordo that add or modify things from the official documents).